interface/forms/eye_mag/SpectacleRx.php

```diff
@@ -289,7 +289,7 @@
     <title><?php echo xlt('Rx Dispensed History'); ?></title>
     <head>
 
-        <?php Header::setupHeader([ 'opener', 'jquery-ui', 'jquery-ui-redmond', 'pure', 'jscolor' ]); ?>
+        <?php Header::setupHeader(['opener', 'pure', 'jscolor']); ?>
 
         <link rel="stylesheet" href="../../forms/<?php echo $form_folder; ?>/css/style.css" type="text/css">
```

Comment:
Hello @bradymiller  and @tywrenn 
In here has relative paths issue, So which one should I change?

https://github.com/openemr/openemr/pull/3497#discussion_r431994305

Reply:
As long that this isn't a link that is used in multiple scripts (which it is not), ok to just keep the relative link here.

Reply:
@bradymiller @stu01509 for consistency sake, since we use globals a lot, we should be using a global variable for the paths here.

Reply:
@tywrenn @stu01509 ,
To change this to absolute path, can use:
```
<link rel="stylesheet" href="<?php echo $GLOBALS['rootdir'] ; ?>/forms/<?php echo $form_folder; ?>/css/style.css" type="text/css">
```